tests/test_create.py

```diff
@@ -1,17 +1,21 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, division, print_function
 
-import pytest
 from contextlib import contextmanager
 from glob import glob
 from logging import getLogger, Handler
 from os.path import exists, isdir, join, relpath
 from shlex import split
 from shutil import rmtree
+import subprocess
+import sys
 from tempfile import gettempdir
 from unittest import TestCase
 from uuid import uuid4
 
+from menuinst.win32 import dirs as win_locations
```

Comment:
FWIW, we'll probably start using [appdirs](https://github.com/ActiveState/appdirs) as a condalib dependency.